go.mod

```diff
@@ -0,0 +1,29 @@
+module github.com/konveyor/asset-generation
+
+go 1.23.3
```

Comment:
The latest version of Go in _go-toolset_ for UBI-9 is 1.22.9.  Do we need 1.23 for anything?

Reply:
Downgraded